src/_data/site.json

```diff
@@ -8,7 +8,7 @@
         "d2fe": "https://boards.greenhouse.io/flowfuse/jobs/5185319004"
     },
     "messaging": {
-        "tagLine": "The Power of Integration at Industrial Scale",
+        "tagLine": "The Power of Integration</br>at Industrial Scale",
```

Comment:
The `</br>` element makes it lose the space between words for the `<title>`
<img width="465" alt="Screenshot 2024-11-25 at 11 25 44" src="https://github.com/user-attachments/assets/cbf4cc7d-c481-4495-ac06-55bb5714c2ff">

Maybe it would be better to change the `max-width` on the `index` file

Reply:
Great! I did try it locally, and it didn’t seem to work for me. I’m glad it was as simple as that.